docs/bigcommerce/installation.mdx

```diff
@@ -0,0 +1,156 @@
+---
+sidebar_position: 1
+title: Installation
+description:
+  This guide explains how to install and configure the Front-Commerce
+  BigCommerce module to start interacting with an existing BigCommerce instance
+  from your application.
+---
+
+<p>{frontMatter.description}</p>
+
+## Configure Front-Commerce to use BigCommerce module
+
+To do so, update your `.front-commerce.js` file as below:
+
+```js title=".front-commerce.js"
+module.exports = {
+  name: "Front-Commerce Skeleton",
+  url: "http://localhost:4000",
+  modules: [
+    "./node_modules/front-commerce/theme-chocolatine",
+    // highlight-next-line
+    "./node_modules/front-commerce/modules/big-commerce",
+    "./src",
+  ],
+  serverModules: [
+    { name: "FrontCommerce", path: "server/modules/front-commerce" },
+    // highlight-next-line
+    { name: "BigCommerce", path: "big-commerce/server/modules" },
+  ],
+  webModules: [
+    { name: "FrontCommerce", path: "front-commerce/src/web" },
+    { name: "ThemeChocolatine", path: "front-commerce/theme-chocolatine/web" },
+    // highlight-start
+    {
+      name: "BigCommerce",
+      path: "front-commerce/modules/big-commerce/web",
+    },
+    {
+      name: "BigCommerceWeb",
+      path: "front-commerce/modules/big-commerce-web/web",
+    },
+    // highlight-end
+    { name: "Skeleton", path: "./src/web" },
+  ],
+};
+```
+
+## Configure the environment
+
+:::note
+
+Those environment variables are described in detail on the
+[Environments variables](/docs/reference/environment-variables#bigcommerce)
+page.
+
+:::
+
+Front-Commerce uses BigCommerce GraphQL and Management API
+
+- Management API variables: `FRONT_COMMERCE_BIG_COMMERCE_ENDPOINT` and
+  `FRONT_COMMERCE_BIG_COMMERCE_AUTH_TOKEN`
+- GraphQL variables: `FRONT_COMMERCE_BIG_COMMERCE_GRAPH_ENDPOINT` and
+  `FRONT_COMMERCE_BIG_COMMERCE_GRAPH_TOKEN`
+
+## [Access to Management API](ttps://developer.bigcommerce.com/docs/ZG9jOjIyMDYxMw-v2-and-v3-rest-api-authentication#obtaining-store-api-credentials)
+
+- In the control panel: `Advanced Settings > API Accounts > Create an API
+  Account > Create V2/V3 API Token``
```

Comment:
```suggestion
  Account > Create V2/V3 API Token`
```
The double backquote breaks the codeblock:
![image](https://user-images.githubusercontent.com/4162177/194560336-96768247-500a-4c6c-ab5e-7e6bb936e462.png)

Reply:
Done in aa0e7e4